pom.xml

```
@@ -6,7 +6,7 @@
 
   <groupId>ir.sahab</groupId>
   <artifactId>smart-commit-kafka-consumer</artifactId>
-  <version>1.4.1-kafka2.7.0-1</version>
+  <version>1.4.1-kafka2.7.0-2</version>
```

```suggestion
  <version>1.4.2-kafka2.7.1</version>
```

1.4.2: Because we are going to apply it in maser branch too
2.7.1: You have changed the Kafka version